.eslintignore

```diff
@@ -0,0 +1,6 @@
+# Ignore minified library files.
+/src/core/js/libraries/*.min.js
```

Comment:
depending on this [pr#2473](https://github.com/adaptlearning/adapt_framework/pull/2473), these paths will need to change to `/src/core/libraries/`